app/src/main/java/com/keylesspalace/tusky/view/ExtendedViewPager.kt

```diff
@@ -0,0 +1,32 @@
+package com.keylesspalace.tusky.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.MotionEvent
+import androidx.viewpager.widget.ViewPager
+
+
+class ExtendedViewPager : ViewPager {
+    private var swipeable = true
+
+    constructor(context: Context) : this(context, null)
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        init(context, attrs)
+    }
```

Comment:
this can be simplified by using a default constructor with overloaded params, e.g. like its done in `ComposeOptionsView`

Reply:
Done